src/pages/programDisaggregations/list/ProgramsList.tsx

```diff
@@ -0,0 +1,182 @@
+import { useAlert } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import {
+    Button,
+    ButtonStrip,
+    Modal,
+    ModalActions,
+    ModalContent,
+    ModalTitle,
+    NoticeBox,
+} from '@dhis2/ui'
+import React, { useCallback, useMemo } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { LinkButton } from '../../../components/LinkButton'
+import { LoadingSpinner } from '../../../components/loading/LoadingSpinner'
+import { ModelSingleSelect } from '../../../components/metadataFormControls/ModelSingleSelect'
+import { useLocationSearchState } from '../../../lib'
+import {
+    PROGRAMS_SELECT_QUERY,
+    useProgramsList,
+    useClearMappingsMutation,
+    getProgramsWithMappings,
+    transformProgramsForSelect,
+    useProgramDeleteModal,
+} from './ProgramListHooks'
+import classes from './ProgramsList.module.css'
+
+export const ProgramsList = () => {
+    const preservedSearchState = useLocationSearchState()
```

Comment:
I appreciate the thought here, but we don't actually make use of the filters in this list? 
Doesn't hurt to keep it if we were to add a search tho.